### PR TITLE
fix(GithubRedirect): use task name instead of task run

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1124

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/pages/GithubRedirectPage.tsx [hunk 1]
```diff
@@ -20,9 +20,9 @@ const GithubRedirectPage = () => {
     workspace
       ? `/workspaces/${workspace.metadata.name}${
           application
-            ? `/applications/${application}${
-                task ? `/taskruns/${task}` : pipelineRun ? `/pipelineruns/${pipelineRun}` : ''
-              }${isLogsTabSelected ? `/logs` : ''}`
+            ? `/applications/${application}${pipelineRun ? `/pipelineruns/${pipelineRun}` : ''}${
+                isLogsTabSelected ? `/logs` : ''
+              }${task ? `?task=${task}` : ''}`
```
Comment (abhinandan13jan) [inline src/pages/GithubRedirectPage.tsx:25]: when no logs tab, why do we add `?task=task` at the end of URL?

Comment (sahil143) [reply src/pages/GithubRedirectPage.tsx:25]: task itself won't be available in the URL if `/logs/` is not present in the redirect URL

**Comment by sahil143** [comment]
retest

**Pull request merged**